react/features/base/sounds/reducer.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+import {assign, ReducerRegistry} from '../redux';
+
+import {
+    _ADD_AUDIO_ELEMENT,
+    _REMOVE_AUDIO_ELEMENT, REGISTER_SOUND, UNREGISTER_SOUND
+} from './actionTypes';
+import type { AudioElement } from '../media';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+type Sound = {
+    audioElement?: AudioElement,
+    soundId: String,
+    src: Any
+}
+
+/**
+ * Initial state of the sounds reducer which is a {@code Map} of globally stored
+ * sounds.
+ *
+ * @type {Map<string, Sound>}
+ */
+const SOUNDS_INITIAL_STATE = new Map();
```

Comment:
Hmm yes I can go with Object here

Reply:
So actually I see that we also use Map in few other reducers and I was thinking it's fine to go with Map as long as it's not being persisted ?

Reply:
Yes, it's fine as long as it's not persisted. It will definitely not persist nicely, but I guess that is not really the plan even for the future, so okay from my side